src/main/java/internet/shop/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,51 @@
+package internet.shop.dao.impl;
+
+import internet.shop.dao.OrderDao;
+import internet.shop.db.Storage;
+import internet.shop.lib.Dao;
+import internet.shop.model.Order;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order create(Order order) {
+        Storage.addOrder(order);
+        return order;
+    }
+
+    @Override
+    public Optional<Order> get(Long id) {
+        return Storage.orders.stream()
+                .filter(x -> x.getId().equals(id)).findFirst();
```

Comment:
usually first letter of the object is used in naming inside of the stream operation (same goes to other methods)
```suggestion
                .filter(o -> o.getId().equals(id)).findFirst();
```